### PR TITLE
net-im/slack-bin: add support for xdg (bug #624612)

### PR DESCRIPTION
**Pull request by SpiderX**
Package-Manager: Portage-2.3.6, Repoman-2.3.1

### CODE REVIEW

**Comment by SpiderX** [comment]
> net-im/slack-bin/slack-bin-2.1.2.ebuild: dosym '/opt/${MY_PN}/${MY_PN}'... could use relative path on line: 69

@mgorny 
To get this warning I have to emerge repoman-9999.
Should I check all further ebuilds with repoman from 9999?

**Comment by mgorny** [comment]
Yes, it is preferable to always use the newest repoman. Nevertheless, I will ping Portage maintainers to do a new release of repoman.

File: net-im/slack-bin/slack-bin-2.1.2.ebuild [hunk 1]
```diff
@@ -66,13 +66,15 @@ src_install() {
 	insinto /opt/${MY_PN}
 	doins -r usr/lib/${MY_PN}/.
 	fperms +x /opt/${MY_PN}/${MY_PN}
-	dosym /opt/${MY_PN}/${MY_PN} /usr/bin/${MY_PN}
+	dosym "${ED}"opt/${MY_PN}/${MY_PN} usr/bin/${MY_PN}
```
Comment (mgorny) [inline net-im/slack-bin/slack-bin-2.1.2.ebuild]: This is 100% wrong. By 'relative symlink' I meant one that starts with appropriate number of `../`s

Comment (SpiderX) [inline net-im/slack-bin/slack-bin-2.1.2.ebuild]: Fixed.

**Pull request closed without merging**